website/js/website.js

```diff
@@ -13,7 +13,7 @@ window.onload = function () {
     aboutCode = window.document.getElementById('context-about');
     aboutCode.innerText = context.aboutString();
 
-    canvas.controller.multiFrameNumber = 8;
+    canvas.controller.multiFrameNumber = 1;
```

Comment:
Is this change intended or is this the result of a debug session?

Reply:
debug session and ok for now, since the default example is always adjusted to its current needs

Reply:
`setInterval()` in `labelrenderer.ts` triggers new frame quite frequently. So if you remove the `setInterval` and change the `multiFrameNumber` back, you got again a very pleasant accumulated frame :)